### PR TITLE
Added support for facets visualization

### PR DESCRIPTION
**Pull request by naveenkumarmarri**

### CODE REVIEW

**Comment by naveenkumarmarri** [comment]
@parente can you please review this?

**Comment by naveenkumarmarri** [comment]
@parente any issue with this PR?

**Comment by parente** [comment]
@naveenkumarmarri I left one comment a few days ago about --sys-prefix that needs to be addressed before we merge.

**Comment by naveenkumarmarri** [comment]
@parente I'm sorry I couldn't find the comment.
Should I change something?

File: scipy-notebook/Dockerfile [hunk 1]
```diff
@@ -40,13 +40,18 @@ RUN conda install --quiet --yes \
...
     'xlrd'  && \
     conda remove --quiet --yes --force qt pyqt && \
     conda clean -tipsy && \
     # Activate ipywidgets extension in the environment that runs the notebook server
     jupyter nbextension enable --py widgetsnbextension --sys-prefix && \
     fix-permissions $CONDA_DIR
 
+RUN git clone https://github.com/PAIR-code/facets.git
+WORKDIR facets
+RUN jupyter nbextension install facets-dist/ --user
```
Comment (jakirkham) [inline scipy-notebook/Dockerfile]: Guessing here, but I think what @parente meant is this should be `jupyter nbextension install --py facets-dist/ --sys-prefix`. Feel free to correct me, @parente.

**Comment by parente** [comment]
That's the spot. For some reason my review comment shows up for me but I guess you can't see it. Hmmm

**Comment by naveenkumarmarri** [comment]
@parente sure I'll make changes and push again
@jakirkham thanks for pointing out

**Comment by parente** [comment]
Sorry, I missed the `--py` in @jakirkham's comment. That should be removed. You just need to replace `--user` with `--sys-prefix`.

**Comment by naveenkumarmarri** [comment]
@parente build is successful
Thanks for pointing it out

**Comment by parente** [comment]
Thanks again for your first contribution here, @naveenkumarmarri, and sorry for the confusion during the review. 🍰

**Pull request merged**